app/views/courses/moss.html.erb

```diff
@@ -1,177 +1,168 @@
 <% @title = "Moss Cheat Checker" %>
 
-<% content_for :javascripts do %>
-  <script type="text/javascript">
-    /**
-     * toggles the "style.display" of the element with the given id.
-     **/
-    function toggleOptions(id) {
-      var show = document.getElementById(id);
-      show.style.display = (show.style.display == "none")? "block" : "none";
-    }
+<h2>Run the Moss Cheat Checker</h2>
 
-    function filterCourses(name) {
-      $(".filterableCourse").each( function() {
-        var course = this;
-        var keywords = name.split(" ");
-        var toShow = true;
-        for (var i=0; i<keywords.length; i++) {
-          if (course.id.toLowerCase().indexOf(keywords[i]) == -1)
-            toShow = false;
-        }
-        $(this).toggle(toShow);
-      });
-    }
-  
-  </script>
-<% end %>
+<% if @course.assessments.empty? %>
+	<ul class="collection" style="text-align: center; font-size: 18px;">
+		<li class="collection-item add">
+			<p>You do not seem to have any assessments in this course to be able to run the Cheat Checker</p>
+			<a title="Install an assessment" class="btn btn-large red darken-3" href=<%= install_assessment_course_assessments_path(@course) %> style="font-size: 18px;">Add an Assessment</a>
```

Comment:
I would suggest wrapping the href attribute in quoates

Reply:
I changed it to use the same line in the index page, but used the url helper for the link to pass the course.